r/R/dataset.R

```diff
@@ -23,17 +23,23 @@
 #' `open_dataset()` to point to a directory of data files and return a
 #' `Dataset`, then use `dplyr` methods to query it.
 #'
-#' @param sources Either:
-#'   * a string path to a directory containing data files
+#' @param sources One of:
+#'   * a string path or URI to a directory containing data files
+#'   * a vector of one or more string paths or URIs to data files
```

Comment:
While "vector" is accurate in the R sense of length one, I think it's good to make explicit that this could be a single file, even if it is technically redundant.

```suggestion
#'   * a string path or URI to a single file
#'   * a character vector of paths or URIs to individual data files
```